docs/dictionary/property/beepSound.lcdoc

```diff
@@ -47,6 +57,10 @@ method specified by the <beepSound> property.
 >*Note:* The iPhone does not have a default system alert sound so if a
 > sounds is required one must be specified by using the <beepSound>
```

Comment:
Hi Andy, Just a small typo here. 'Sound' not 'sounds' and not need for the 'by'. Should read:
> so if a sound is required one must be specified using the \<beepSound\> property.